### DISCUSSION TITLE
gate-legato-pick up clip launching mode

### DISCUSSION THREAD

**Discussion by prtrjan** [helgoboss/helgobox#1125] [Ideas]
<img width="1090" alt="Screenshot 2024-08-25 at 13 29 05" src="https://github.com/user-attachments/assets/f7db7882-34d5-4eec-a6de-a7e6184ac305">

I found a concise description of possible clip launching modes. I would love to see a gated/pick-up mode in playtime.

source: atom 2 piano roll manual (ios midi app)

**Comment by prtrjan** [comment]
in pick up/gate mode (clip runs silently in lock-sync with host until launched/unmuted) for audio clips: allow to set attack/release times (or use microfades to fade in/out to avoid clicks)

**Comment by helgoboss** [comment]
Sort of a duplicate of #1038. That concise description above is nice. I will link this discussion there but close it.

**Reply by prtrjan** [reply]
Yes, certainly compatible with the other discussion. Do you think it a good idea to incorporate attack/release/microfades to avoid clicking when in gate mode (for audio items)?

**Reply by helgoboss** [reply]
Yes, definitely. Playtime already uses microfades in lots of places. Gate mode wouldn't be an exception.

**Reply by prtrjan** [reply]
hadn't realised that!

**Reply by helgoboss** [reply]
I guess this is one of those features people would only realize if they are not there ;) Because it would click as hell.
